Snowflake.Data.Tests/Util/PoolConfig.cs

```diff
@@ -16,15 +17,17 @@ class PoolConfig
 
         public PoolConfig()
         {
-            _maxPoolSize = SnowflakeDbConnectionPool.GetMaxPoolSize();
-            _timeout = SnowflakeDbConnectionPool.GetTimeout();
-            _pooling = SnowflakeDbConnectionPool.GetPooling();
-            _connectionPoolType = SnowflakeDbConnectionPool.GetConnectionPoolVersion();
+            _maxPoolSize = SFSessionHttpClientProperties.DefaultMaxPoolSize;
+            _timeout = (long) SFSessionHttpClientProperties.DefaultExpirationTimeout.TotalSeconds;
+            _pooling = SFSessionHttpClientProperties.DefaultPoolingEnabled;
+            _connectionPoolType = SnowflakeDbConnectionPool.DefaultConnectionPoolType;
         }
 
         public void Reset()
         {
             SnowflakeDbConnectionPool.SetConnectionPoolVersion(_connectionPoolType);
+            if (_connectionPoolType == ConnectionPoolType.MultipleConnectionPool)
+                return; // for multiple connection pool setting parameters for all the pools doesn't work by design
```

Comment:
minor but it could be visible as a debug information so to track improper use of the interface within tests

Reply:
but the log level in tests is warn so it won't be visible either way